seirmo/apps/seir_simulation_app.py

```diff
@@ -68,26 +69,34 @@
  infecting rate of the disease, many models are used to characterise the
  disease, such as the SEIR model and the branching processes model.  
   
-SEIR model is a deterministic model and has been used to model infectious
- disease.  """
+SEIR model is a deterministic model. A deterministic model means that the simulation will always 
+be the same as long as the parameters are the same. This SEIR model has been used to model 
+infectious disease.  """
 
 # Model description
 description = """
 The SEIR model is a model of Ordinary Differential Equations (ODEs). 
```

Comment:
If we want this to be accessible to students maybe don't start with ODEs? E.g.

The SEIR model assigns individuals in a population to four distinct disease stages: susceptible (S), infected but not infectious (E), infectious (I) and recovered (R). Susceptible individuals can become infected, then infectious and ultimately recover. Diagrammatically this may be illustrated by the figure below.


More formally the SEIR model is expressed in terms of a system of ordinary differential equations, whose transition rates between the states S, E, I and R are controlled by rate parameters. The *incubation period* and the *infection period*. The *reproduction number* measures the number of *infection rate* indicates... (and so on)

What do you think?

Reply:
We did not use incubation period, infection period and reproduction number in the plot and sliders, I think it would be quite confusing for students. We could add it in the future?